src/main/java/Duke.java

```diff
@@ -8,21 +9,32 @@ public static void main(String[] args) {
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
-
         greet();
+        
         Scanner scanner = new Scanner(System.in);     
+        ArrayList<String> Tasks = new ArrayList<String>();
 
         user_active: while (true) {
             String usercommand = scanner.nextLine();
```

Comment:
You may consider writing your variable names in camelCase? So perhaps userCommand instead of usercommand